src/main/java/org/mutabilitydetector/checkers/MutableTypeToFieldChecker.java

```diff
@@ -169,9 +174,8 @@ private void checkIfClassIsMutable(FieldInsnNode fieldInsnNode, Type typeAssigne
                     setAssigningToGenericFieldResult(fieldName, fieldLocation);
                     break;
                 }
-
-                MutabilityLookup mutabilityLookup = mutableTypeInfo.resultOf(dotted(ownerClass), assignedToField, analysisInProgress);
-
+                MutabilityLookup mutabilityLookup = mutableTypeInfo.resultOf(dotted(ownerClass), assignedToField,analysisInProgress);
+                
```

Comment:
This change can be reverted?

Reply:
Nope this cant be reverted because of code changes happened above this in method visitField